### PR TITLE
Support PureScript v0.15

### PR DESCRIPTION
**Pull request by prikhi**
Switch from bower + pulp to spago, update the example to support
PureScript v0.15.

### CODE REVIEW

**Comment by prikhi** [comment]
Can this come with a new release so I don't have to tie my package list to a commit?

Would love having jest, markdown-it, html-parser-halogen, & this repo in spago package sets if you'd be into that, these `packages.dhall` are getting long :grin:

File: spago.dhall [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+You can edit this file as you like.
+
+Need help? See the following resources:
+- Spago documentation: https://github.com/purescript/spago
+- Dhall language tour: https://docs.dhall-lang.org/tutorials/Language-Tour.html
+
+When creating a new Spago project, you can use
+`spago init --no-comments` or `spago init -C`
+to generate this file without the comments in this block.
+-}
```
Comment (rnons) [inline spago.dhall]: If you don't mind, I prefer to remove those generated comments.

**Review by rnons — APPROVED** [review]
Looks good, thanks.

File: spago.dhall (hunk 1 above)
Comment (prikhi) [reply spago.dhall]: Fixed

**Pull request merged**